build/msw/wx_custom_build.vcxproj

```diff
@@ -131,6 +131,11 @@
     <TargetName Condition="'$(Configuration)|$(Platform)'=='DLL Release|Win32'">$(wxToolkitDllNamePrefix)$(ProjectName)$(wxToolkitDllNameSuffix)</TargetName>
     <TargetName Condition="'$(Configuration)|$(Platform)'=='DLL Release|x64'">$(wxToolkitDllNamePrefix)$(ProjectName)$(wxToolkitDllNameSuffix)</TargetName>
   </PropertyGroup>
+  <ItemDefinitionGroup Condition="!Exists('..\..\include\wx\msw\setup.h')">
+    <PreBuildEvent>
+      <Command>copy ..\..\include\wx\msw\setup0.h ..\..\include\wx\msw\setup.h /Y</Command>
```

Comment:
Why `/Y`? Shouldn't there already be no confirmation if/as the destination file doesn't exist?